htdocs/js/smr15.js

```diff
@@ -102,8 +102,57 @@ function showRaceInfo(select) {
 	var desc = document.getElementById('race_descr');
 	$(desc).children().addClass('hide');
 	$(".race_descr" + race_id, desc).removeClass('hide');
+	graph_by_plotly(race_id);
 }
 
+// Used by game_join.php 
+function graph_by_plotly(race_id) {
+
+	var races = [["[1.0,1.0,1.0,1.0,1.0]","'Neutral'","'#FFD800'"],
+	[[1.0,1.0,1.0,1.0,1.0],'Neutral','#FFD800'],
+	[[5.2, 10.0, 5.8, 8.1, 5.2],'Alaskant','#FF00FF'],
+	[[8.3, 5.0, 10.0, 3.3, 8.3],'Creonti','#FF8000'],
+	[[9.5, 4.9, 9.5, 7.7, 9.5],'Human','#0000FF'],
+	[[8.0, 6.1, 9.7, 10.0, 8.0],'Ik\'Thorne','#BFBFFF'],
+	[[8.1, 4.7, 8.8, 4.3, 8.1],'Salvene','#00AA00'],
+	[[10.0, 5.7, 8.7, 9.1, 10.0],'Thevian','#800000'],
+	[[7.2, 6.3, 7.9, 6.2, 7.2],'WQ Human','#804040'],
+	[[9.5, 4.9, 8.0, 5.1, 9.5],'Nijarin','#FF8080']];
+	data = [
+		{
+			type: 'scatterpolar',
+			r: races[race_id][0],
+			theta: ['Hunting','Trading','Combat', 'Utility', 'Hunting'],
```

Comment:
Why is "Hunting" in this array twice? Can this be fixed by just omitting one of them, or is this a hack to solve some other issue?

(Also, spaces between items.)

Reply:
This is required because last plot is from Utility back to Hunting.  It is how Plotly works. Omitting last plot leaves a missing line.